library/java/io/envoyproxy/envoymobile/engine/types/EnvoyStreamIntel.java

```diff
@@ -18,4 +18,8 @@
    * The number of internal attempts to carry out a request/operation.
    */
   public long getAttemptCount();
+  /*
+   * The number of bytes received from upstream.
```

Comment:
update commments?

Reply:
Comments updates, and also using the same names everywhere. Thanks.